### PR TITLE
gcc: update to 8.3

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: srcpkgs/cross-aarch64-linux-musl/template [hunk 1]
```diff
@@ -22,18 +22,17 @@ distfiles="
...
- 
 
 lib32disabled=yes
 nocross=yes
 nopie=yes
 nodebug=yes
 create_wrksrc=yes
 
-only_for_archs="x86_64 x86_64-musl ppc64le"
+archs="x86_64 x86_64-musl ppc64le"
```
Comment (maxice8) [inline srcpkgs/cross-aarch64-linux-musl/template]: can be x86_64*

File: srcpkgs/cross-powerpc64-linux-musl/template [hunk 2]
```diff
@@ -34,7 +34,7 @@ makedepends="zlib-devel gmp-devel mpfr-devel libmpc-devel isl15-devel"
 nostrip_files="libcaf_single.a libgcc.a libgcov.a libgcc_eh.a
  libgnarl_pic.a libgnarl.a libgnat_pic.a libgnat.a"
 depends="${pkgname}-libc-${version}_${revision}"
-only_for_archs="x86_64 x86_64-musl ppc64le"
+archs="x86_64 x86_64-musl ppc64le"
```
Comment (maxice8) [inline srcpkgs/cross-powerpc64-linux-musl/template]: likewise

File: srcpkgs/cross-powerpc64le-linux-musl/template [hunk 3]
```diff
@@ -34,7 +34,7 @@ makedepends="zlib-devel gmp-devel mpfr-devel libmpc-devel isl15-devel"
 nostrip_files="libcaf_single.a libgcc.a libgcov.a libgcc_eh.a
  libgnarl_pic.a libgnarl.a libgnat_pic.a libgnat.a"
 depends="${pkgname}-libc-${version}_${revision}"
-only_for_archs="x86_64 x86_64-musl ppc64le"
+archs="x86_64 x86_64-musl ppc64le"
```
Comment (maxice8) [inline srcpkgs/cross-powerpc64le-linux-musl/template]: and here too

File: srcpkgs/cross-x86_64-linux-musl/template [hunk 4]
```diff
@@ -35,7 +35,7 @@ makedepends="zlib-devel gmp-devel mpfr-devel libmpc-devel isl15-devel"
 nostrip_files="libcaf_single.a libgcc.a libgcov.a libgcc_eh.a
  libgnarl_pic.a libgnarl.a libgnat_pic.a libgnat.a"
 depends="${pkgname}-libc-${version}_${revision}"
-only_for_archs="i686 i686-musl x86_64"
+archs="i686 i686-musl x86_64"
```
Comment (maxice8) [inline srcpkgs/cross-x86_64-linux-musl/template]: can be done with i686 here

File: srcpkgs/gcc/template [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 _gmp_version=6.1.2
 _mpfr_version=4.0.1
 _mpc_version=1.1.0
 _isl_version=0.19
 
 pkgname=gcc
 version=${_majorver}.0
-revision=9
+revision=1
 short_desc="The GNU C Compiler"
```
Comment (maxice8) [inline srcpkgs/gcc/template]: GNU C Compiler

Comment (jnbr) [reply srcpkgs/gcc/template]: changed

**Comment by jnbr** [comment]
I've built a few hundred packages on each of x86_64, x86_64-glibc, i686 armv7 and aarch64-musl so far.
Except qt (#9216) and qt5 (#9215) I haven't seen any breakage yet.

**Comment by pullmoll** [comment]
The step from 8.2.0 to 8.3.0 is probably easier than previous major version jumps.
IMO this can be merged.

**Comment by jnbr** [comment]
merged

**Comment by pullmoll** [comment]
For me cross meson now causes trouble / fails to build packages like e.g. `gobject-introspection` even for GNU libc arches. That used to work before while I can't tell for sure it happens since the gcc update or is caused by something else.

**Comment by jnbr** [comment]
looks like a problem with the latest changes in `common/build-style/meson.sh` @maxice8

**Pull request closed without merging**